metabase/metabase
# asynchronously load libraries for png and pdf exports

Epic https://github.com/metabase/metabase/issues/41032

### Description

Reduces the size of the vendor bundle by extracting heavy libraries used for pdf and png exports which should not block the initial app loading.

Before
<img width="319" alt="Screenshot 2024-04-04 at 11 19 37 AM" src="https://github.com/metabase/metabase/assets/14301985/32149389-b82b-4b32-a55b-2dceb3c0c9df">

After
<img width="322" alt="Screenshot 2024-04-04 at 11 13 33 AM" src="https://github.com/metabase/metabase/assets/14301985/16550f4a-0bd8-42f7-a3e4-0796c5fa683f">

### How to verify

- Ensure png and pdf exports are not broken
- Using webpack bundle analyzer you can confirm the `vendor` chunk size

### Demo

<video src="https://github.com/metabase/metabase/assets/14301985/bd19b9b3-2150-434c-a2a6-3348d5a1b8cf" />
